app/src/components/bcgov/ButtonStyles.ts

```diff
@@ -0,0 +1,37 @@
+import { bcgov } from "../../constants/colours";
+import { normalFont } from "../../constants/fonts";
+
+const commonButtonStyle = {
+    ...normalFont,
+    borderRadius: '4px',
+    padding: '12px 24px',
+    textAlign: 'center',
+    textDecoration: 'none',
+    display: 'block',
+    fontSize: '14px',
+    fontWeight: '700',
+    letterSpacing: '1px',
+    cursor: 'pointer',
+  '&:hover': {
```

Comment:
this seems like you're missing an indent here? looks weird to me.

Reply:
There's an extra indent for the non-hover styling. Was trying to keep it visually identifiable, but can align it all if you think that's better.

Reply:
I typically use an extra line break. Indentation is finicky and most of the time our prettier config will fix indentations like this.

Reply:
Updated to use consistent indentation with line break instead.